components/webdriver_server/actions.rs

```diff
@@ -156,22 +169,32 @@ impl Handler {
                             match action {
                                 PointerAction::Cancel => (),
                                 PointerAction::Down(action) => {
-                                    self.dispatch_pointerdown_action(&source_id, &action)
+                                    self.dispatch_pointerdown_action(&source_id, &action)?
                                 },
-                                PointerAction::Move(_action) => (),
+                                PointerAction::Move(action) => self.dispatch_pointermove_action(
+                                    &source_id,
+                                    &action,
+                                    tick_duration,
+                                )?,
                                 PointerAction::Up(action) => {
-                                    self.dispatch_pointerup_action(&source_id, &action)
+                                    self.dispatch_pointerup_action(&source_id, &action)?
                                 },
                             }
                         },
                     }
                 }
             },
         }
+
+        Ok(())
     }
 
     // https://w3c.github.io/webdriver/#dfn-dispatch-a-keydown-action
-    fn dispatch_keydown_action(&mut self, source_id: &str, action: &KeyDownAction) {
+    fn dispatch_keydown_action(
+        &mut self,
+        source_id: &str,
+        action: &KeyDownAction,
+    ) -> Result<(), ErrorStatus> {
```

Comment:
Why this change if no errors can occur? Same question for the other dispatch methods that were changed in the same way.